dataland-frontend/tests/e2e/specs/sfdr/DataIntegrity.ts

```diff
@@ -0,0 +1,156 @@
+            return { companyId: companyId, dataId: dataMetaInformation.dataId };
+          });
+        });
+      });
+    }
+
+    /**
+     * Parses the year from a date string
+     *
+     * @param sfdrDate date to parse
+     * @returns the year from the date as string
+     */
+    function getYearFromSfdrDate(sfdrDate: string): string {
+      return new Date(sfdrDate).getFullYear().toString();
+    }
+
+    /**
+     * Gets an SFDR dataset based on the provided data ID and parses the Fiscal Year End from its date field
+     *
+     * @param token The API bearer token to use
+     * @param dataId The data ID of an SFDR dataset
+     * @returns the year from the Fiscal Year End value of the SFDR dataset as string
+     */
+    async function getFiscalYearEndOfSfdrDataSet(token: string, dataId: string): Promise<string> {
+      const response = await new SfdrDataControllerApi(
+        new Configuration({ accessToken: token })
+      ).getCompanyAssociatedSfdrData(dataId);
+      const sfdrData = response.data.data as SfdrData;
+      if (sfdrData) {
+        const fiscalYearEndAsString = sfdrData.social!.general!.fiscalYearEnd as string;
+        return getYearFromSfdrDate(fiscalYearEndAsString);
+      } else {
+        throw Error(`No Sfdr dataset could be retrieved for the provided dataId ${dataId}`);
+      }
+    }
+
+    it("Check Sfdr view page for company with one Sfdr data set", () => {
+      const preparedFixture = getPreparedFixture("company-with-one-sfdr-data-set", preparedFixtures);
+      const companyInformation = preparedFixture.companyInformation;
+      const sfdrData = preparedFixture.t;
+
+      getKeycloakToken(uploader_name, uploader_pw).then(async (token: string) => {
+        return uploadCompanyAndSfdrDataViaApi(token, companyInformation, sfdrData).then((uploadIds) => {
+          cy.intercept("**/api/data/sfdr/company/*").as("retrieveSfdrData");
+          cy.visitAndCheckAppMount(`/companies/${uploadIds.companyId}/frameworks/sfdr`);
+          cy.wait("@retrieveSfdrData", { timeout: Cypress.env("medium_timeout_in_ms") as number }).then(() => {
+            cy.get(`h1`).should("contain", companyInformation.companyName);
+
+            cy.get("table.p-datatable-table")
+              .find(`span:contains(${sfdrData.social!.general!.fiscalYearEnd!})`)
+              .should("exist");
+
+            cy.get("button.p-row-toggler").eq(0).click();
+            cy.get("table.p-datatable-table")
+              .find(`span:contains(${sfdrData.social!.general!.fiscalYearEnd!})`)
+              .should("not.exist");
+
+            cy.get("button.p-row-toggler").eq(0).click();
+            cy.get("table.p-datatable-table")
+              .find(`span:contains(${sfdrData.social!.general!.fiscalYearEnd!})`)
+              .should("exist");
+          });
+        });
+      });
+
+      cy.get("table.p-datatable-table").find(`span.link`).click();
+      cy.request("GET", "/test_pdf.pdf").then((response) => {
```

Comment:
as discussed lets remove the files until we actually have the document upload included